roundabout-core/src/main/java/at/fhv/itm3/s2/roundabout/Sink.java

```diff
@@ -26,12 +29,22 @@ public double getLength() {
 
     @Override
     public void addCar(ICar car) {
+        if (carQueue == null) {
```

Comment:
this car queue cannot be null because the list is always initialized in the constructor